content/topics/Analyze/causal-inference/matching/matching.md

```diff
@@ -0,0 +1,307 @@
+description: "Matching is used to create comparable groups in observational studies, helping to mitigate the effects of confounding variables and estimate causal effects."
+keywords: "matching, causal, inference, effect,regression, R, exact, approximate"
+draft: false
+weight: 1
+author: "Valerie Vossen"
+aliases:
+  - /exact-matching
+  - /matching
+---
+
+## Overview
+
+Randomization is a fundamental principle in experimental design, aiming to have a good counterfactual and ensuring the treatment and control groups are similar except for being treated. However, in cases where treatment is not randomly assigned, confounding variables can bias the estimated causal effect.
+
+Matching offers an alternative approach by basically creating an artificial counterfactual. This is done by pairing individuals or units based on specific observable characteristics to create comparable groups. This article provides a comprehensive introduction to matching, focusing on both the theory behind the method and practical applications.
+
+## Exact and approximate matching
+
+Exact matching involves pairing individuals who share **identical characteristics**. This requires the observable characteristic on which pairing happens to be a binary variable. Also ideally, the control group has many observations at each distinct value of the binary variable the observations are matched on. 
+
+In contrast, [approximate matching](/approximate-matching) allows for some degree of flexibility and pairs on similar but not identical characteristics. 
+
+
+## Identifying assumptions
+
+{{% summary %}}
+
+Two identifying assumptions should hold for exact matching to be valid:
+
+1. *Conditional Independence Assumption:* Given observed covariates *X*, there are no systematic differences between treated and untreated units.
+<br>
+<br>
+
+2. *Overlapping support*: Conditional on *X*, there are treated and untreated units.
+
+{{% /summary %}}
+
+
+### 1. Conditional Independence Assumption (CIA)
+
+The Conditional Independence Assumption is the primary assumption underlying matching. It states that **once we account for the observed characteristics of units or individuals, there should be no systematic differences in potential outcomes between the treatment and control groups**. 
+
+Mathematically, this assumption is expressed as follows:
+
+{{<katex>}}
+(Y_{0}, Y_{1}) \perp T \:|\: X
+{{</katex>}}
+
+<br>
+<br>
+
+Where $Y_{0}$ and $Y_{1}$ are the potential outcomes in respectively the control and treatment group, which are independent of treatment assignment $T$ for each value of the observed covariates $X$. The $X$ is/are the binary variable(s) on which the observations are matched.
+
+{{% warning %}}
+
+This assumption also means there are no *unobservable* characteristics differing between the groups, affecting treatment likelihood or effects. This cannot be directly tested. However, there are some methods to assess the plausibility of the assumption. 
```

Comment:
You could add that the assumption means that when we divide up the sample by X, then for each value of X, we have that the variation in D is as good as random. Sometimes, one knows that this is the case. A famous example is "Project STAR" where a treatment was assigned randomly within schools, but with a different likelihood for different schools. So school identity is in X.